landoapi/transactions.py

```diff
@@ -14,7 +20,10 @@
 
 
 def transaction_search(
-    phabricator, object_identifier, transaction_phids=None, limit=100
+    phabricator: PhabricatorClient,
+    object_identifier: str,
+    transaction_phids: Optional[list[str]] = None,
```

Comment:
What version of Python does lando run on? depending on that might need to change to `List[str]`, but other than that looks goood.

Reply:
Lando runs on Python 3.9 at the moment. You can find the Python version in the [Dockerfile for Lando](https://github.com/mozilla-conduit/lando-api/blob/4244a2078bbe98187fcec727f8d43ee939c058aa/Dockerfile#L5) as we build Lando from the `python` image repository.